xdsl/parser/attribute_parser.py

```diff
@@ -600,6 +601,14 @@ def _parse_tensor_attrs(self) -> AnyTensorType | AnyUnrankedTensorType:
 
         return TensorType(type, shape)
 
+    def _parse_tuple_attrs(self) -> TupleType:
+        params = self.parse_optional_undelimited_comma_separated_list(
+            self.parse_optional_attribute, self.parse_attribute
+        )
+        if params is None:
+            params = []
```

Comment:
MLIR doesn't accept `tuple` (with no `<>`) so I imagine we shouldn't either.

Reply:
I thought this would accept the angle brackets being omitted but I seem to be wrong